Dockerfile

```diff
@@ -1,32 +1,47 @@
-FROM node:8.9.3
+FROM node:8.15.1
```

Comment:
8.15.1 will not build as nexe does not support it.

https://github.com/nexe/nexe/releases